src/Adapter/Form/ChoiceProvider/TaxRuleGroupChoiceProvider.php

```diff
@@ -61,7 +61,7 @@ public function getChoicesAttributes(): array
             }
 
             $attrs[$rule['name']] = [
-                'data-tax-rate' => !empty($rule['rate']) ? $rule['rate'] : null,
+                'data-tax-rate' => !empty($rule['rate']) ? $rule['rate'] / 100 : 0,
```

Comment:
The idea is to have some convention across the whole, all the other tax rates are provided in their "float" value (between 0 and 1) instead of the percent one (between 1 and 100) because anyway the JS had to perform this operation to compute the tax ratio.

One is no better than the other in the end, but I thought it was probably better to have a common format instead of values as float and other as percent. And even if this provider is shared by two form types there is only the product one which uses the `getChoicesAttributes` method so no impact outside of this context.

Reply:
Oh but I just noticed it was I who developed the other code that was not in percentage value It's recent code so it's still modifiable, I'm gonna update it to use the percent value everywhere